src/components/pages/InstructorBooking/InstructorBookingCards.js

```diff
@@ -22,10 +23,11 @@ const InstructorBookingCards = props => {
   const [currentCourses, setCurrentCourses] = useState();
 
   const dispatch = useDispatch();
-
+  const { authState } = useOktaAuth();
+  const { idToken } = authState;
```

Comment:
nice and clean. id token is cleanly destructured from authState, which is obtained via an instance of useOktaAuth. Then utilized in dispatch upon component mount in useEffect hook (typ.)